containers/docker-compose.yaml

```diff
@@ -98,7 +98,6 @@ services:
       - ./quarterdeck/keys:/data/keys
       - ./quarterdeck/emails:/data/emails
     environment:
-      - QUARTERDECK_SENDGRID_TESTING=true
```

Comment:
Fixing a duplicate variable which was preventing me from building the containers.